src/Model/Look.php

```diff
@@ -23,6 +23,12 @@ class Look implements LookInterface
 
     protected ?string $code = null;
 
+    protected float $percentageDiscount = 0.0;
```

Comment:
```suggestion
    protected float $discount = 0.0;
```

Wouldn't this be enough?